src/platform/backends/lxd/lxd_virtual_machine_factory.h

```diff
@@ -52,6 +52,11 @@ class LXDVirtualMachineFactory : public BaseVirtualMachineFactory
 
     std::vector<NetworkInterfaceInfo> networks() const override;
 
+    std::string bridge_name_for(const std::string& iface_name) const override
```

Comment:
Hey @luis4a0, once networking is implemented for QEMU, this will be necessary there too. Furthermore, this duplicates functionality that exists in [`mp::Backend::create_bridge_with`](https://github.com/canonical/multipass/blob/22ca45f1d6ab3ffacac0c6bc3d0bc575675fc24e/src/platform/backends/shared/linux/backend_utils.cpp#L201), where we also derive the bridge name based on a constant maximum string length. 

So, could this:

1. be implemented in `backend_utils.cpp`, with the implementation here forwarding there (as done for [`mp::LXDVirtualMachineFactory::create_bridge_with`](https://github.com/canonical/multipass/blob/f1ab5b20f0c521e74fd4eecb9340d88c2754db2a/src/platform/backends/lxd/lxd_virtual_machine_factory.cpp#L260))?
2. use the existing constant in backend_utils?
3. replace the current implementation in `create_bridge_with` (i.e. `create_bridge_with` would no longer derive the bridge name in its own code, but instead call this utility function)?

Please let me know if there are difficulties with that.

Reply:
Hey @ricab! Done, I was thinking in something like that after discussing about it earlier.

Reply:
Great, thanks!